(fix|COS-3461): Contract social link is set to organization social link value

Kasia9090 opened the pull request:
…

## Proposed changes

<!---Describe the big picture of your changes here.  If it fixes a bug or resolves a feature request, be sure to link that issue!--->

## Changes

What types of changes does your code introduce?  _Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build related changes
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (please describe below)

## Additional context



<!-- This is an auto-generated comment: release notes by coderabbit.ai -->

## Summary by CodeRabbit

- **New Features**
  - Introduced a new `FormSocialInput` component for managing social links.
  - Added functionality to add, update, and remove social links for organizations.
  - Implemented a `SocialIcon` component to render social media icons based on URLs.

- **Enhancements**
  - Replaced `SocialIconInput` with `FormSocialInput` for better social link management.
  - Improved handling of social media URLs with the `isKnownUrl` function.

<!-- end of auto-generated comment: release notes by coderabbit.ai -->

Kasia9090 committed:
(fix|COS-3461): Contract social link is set to organization social link value

umihai29 reviewed (commented):
Since you already implemented the store on Contracts why we can just use the component I refactored in the AboutTab regarding this SocialInput ?

Kasia9090 commented:
> Since you already implemented the store on Contracts why we can just use the component I refactored in the AboutTab regarding this SocialInput ?


I implemented store for contracts not contacts, so they have no overlap here
